assets/js/modules/analytics-4/datastore/__fixtures__/report.json

```diff
@@ -0,0 +1,34 @@
+[
+	{
+		"columnHeader": {
+			"dimensions": null,
+			"metricHeader": {
+				"metricHeaderEntries": [
+					{
+						"name": "ga:sessionDuration",
```

Comment:
This looks the same as https://github.com/google/site-kit-wp/blob/e8285aa24b8b271b04a4f35abf47d05b838ed9c9/assets/js/modules/analytics/datastore/__fixtures__/report.json which is UA-specific. If this isn't used yet we can probably just remove it, as we will be using the tool in #6174 for cases where we would have used a fixture in the past. Fixtures make a bit more sense for the other structures here, but report data requires a lot of different cases where one fixture wouldn't be very useful.

Reply:
For now, I updated it to match the [Analytics-4 report](https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/batchRunReports#runreportrequest). So later we can switch to the tool from #6174.